### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by mcwongmc** [444B/streamlit-analytics2#97]
**Describe the bug**
- When trying to reset analytics in /?analytics=on, the reset button will not apear after choosing the "I'm absolutely sure that I want to reset the results".

- I think the root cause is a typo in src/streamlit_analytics2/display.py 
- line 153-160
```
            reset_prompt = st.selectbox(
                "Continue?",
                [
                    "No idea what I'm doing here",
                    "I'm absolutely sure that I want to reset the results",
                ],
            )
            if reset_prompt == "I'm sure that I want to reset the results":
```
- The if statement in 160 has a typo that cannot match the selectbox in 157.

**Expected behavior**
- The button "Click here to reset" should appear

**To Reproduce**
1. Go to '/?analytics=on' path > Danger Zone
2. Click on 'I'm absolutely sure that I want to reset the results'
3. See error

**Software Versions**
Please fill in the relevant field:
- streamlit_analytics2 : 0.7.5
- streamlit : 1.35.0
- Python3 : 3.9-slim

**Screenshots**
- 

**Additional context**
- code line:
https://github.com/444B/streamlit-analytics2/blob/fe598d1814b15ff4a848e476aa053e418887fa74/src/streamlit_analytics2/display.py#L160

**Comment by 444B** [comment]
@mcwongmc 
Thank you for raising this!
This is a really high quality bug and I can confirm that this is unexpected behavior in the current prod packages. I'll provide an RCA after a fix

I am away from my laptop but within next 24h I will release a new version 0.7.6 with a patch

**Comment by 444B** [comment]
@mcwongmc 
Thank you again for raising this, this has been resolved in release 0.7.6, available now on [pypi](https://pypi.org/project/streamlit-analytics2/)  

The root cause (as you correctly identified) was the removal on the word "absolutely" on current line 160.
This removal was part of a flake8 / Black workflow that removed the word as it picked it up as redundant.
Thanks for identifying this!
